lab-judy/lib/error-middleware.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+const createError = require('http-errors');
+const debug = require('debug')('note:error-middleware');
```

Comment:
You should name your debug module after your model -- so it'll be like 'person:storage'.